docs/user_guide/index.rst

```diff
@@ -38,7 +38,7 @@ explained below.
 Series
 ------
 A series is a one-dimensional sequence with numeric values (64-bit float) and
-unique indicies (64-bit integer). (Consequently, each numeric value is natively
+unique indicies (64-bit integer) (Consequently, each numeric value is natively
```

Comment:
I'd make the sentence starting with Consequently a regular sentence (not in brackets): "A series is a one-dimensional sequence with numeric values (64-bit float) and unique indices (64-bit integer). Consequently, each numeric value is natively represented with 128-bits."

Also there's a typo in "indicies"